lib/geoengineer/templates/json_rest_api.rb

```diff
@@ -71,32 +71,21 @@ def create_rest_integrations(api_resources, params)
         self["type"] = "AWS_PROXY"
         integration_http_method "POST" # ALWAYS POST TO LAMBDAS
         uri "#{api_gateway_arn}:lambda:path/2015-03-31/#{invocation_arn}"
+        depends_on [api_methods.values].flatten.map(&:terraform_name)
       }
     end
     api_integrations
   end
 
   def http_method_response_mappings(api_resources, params)
-    https_methods = params[:methods].values.map { |m| m[:method] }.uniq
-
-    response_mappings = {}
-    https_methods.each do |m|
-      response_mappings["#{m}_success"] = {
-        status: "200",
-        method: m
-      }
+    params[:methods].values.each do |meth|
```

Comment:
Do we not need the uniqueing on method anymore?

Reply:
I am unsure why, but when I do, it breaks terraform, and when I don't it doesnt matter.